ssl/t1_lib.c

```diff
@@ -844,6 +849,7 @@ static int rsa_pss_check_min_key_size(const RSA *rsa, const SIGALG_LOOKUP *lu)
  */
 static const SIGALG_LOOKUP *tls1_get_legacy_sigalg(const SSL *s, int idx)
 {
+    int start_idx = idx;
```

Comment:
Style nit: we prefer a blank line between declarations and the rest of the code.